### PR TITLE
rm: add Korean translation

### PR DESCRIPTION
**Pull request by luhavis**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/linux/rm.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# rm
+
+> 파일 또는 디렉터리 삭제.
+> 같이 보기: `rmdir`.
+> 더 많은 정보: <https://www.gnu.org/software/coreutils/rm>.
+
+- 특정 파일 삭제:
+
+`rm {{경로/파일1 경로/파일2 ...}}`
```
Comment (kbdharun) [inline pages.ko/linux/rm.md:9]:
```suggestion
`rm {{경로/대상/파일1 경로/대상/파일2 ...}}`
```

File: pages.ko/linux/rm.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`rm --interactive {{경로/파일1 경로/파일2 ...}}`
+
+- 특정 파일을 삭제하고 삭제한 파일 정보를 출력:
+
+`rm --verbose {{경로/파일1 경로/파일2 ...}}`
+
+- 특정 파일 및 디렉터리를 재귀적으로 삭제:
+
+`rm --recursive {{경로/파일1_또는_디렉터리1 경로/파일2_또는_디렉터리2 ...}}`
```
Comment (kbdharun) [inline pages.ko/linux/rm.md:25]:
```suggestion
`rm --force {{경로/대상/파일1 경로/대상/파일2 ...}}`

- 특정 파일을 삭제하기 전에 대화형 메시지를 표시하여 특정 파일을 삭제:

`rm --interactive {{경로/대상/파일1 경로/대상/파일2 ...}}`

- 특정 파일을 삭제하고 삭제한 파일 정보를 출력:

`rm --verbose {{경로/대상/파일1 경로/대상/파일2 ...}}`

- 특정 파일 및 디렉터리를 재귀적으로 삭제:

`rm --recursive {{경로/대상/파일_또는_폴더1 경로/대상/파일_또는_폴더2 ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

Note: We have standard translations for generic path placeholders documented in https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

**Review by kbdharun — APPROVED** [review]

**Comment by luhavis** [comment]
@kbdharun 
Thank you. i fixed it you suggested.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**